encrypted-media-respec.html

```diff
@@ -3840,7 +3851,7 @@
             <li><p>Origins visited (via stored or in-memory data, permissions, etc.)</p></li>
             <li><p>Content viewed (via stored or in-memory licenses, keys, key IDs, records of key usage, records of license destruction, etc.)</p></li>
           </ul>
-          <p>This specification presents a specific concern because such information is commonly stored outside the user agent (and its profile storage), often in the CDM.</p>
+          <p>This specification presents a specific concern because such information is commonly stored outside the user agent (and its <a def-id="user-agent-profile"></a> storage), often in the CDM.</p>
```

Comment:
Note: "its" here implies that whatever we define is a UA thing. Perhaps this should be "the," or "the associated," "the corresponding."